.gitignore

```diff
@@ -0,0 +1,15 @@
+.#*
+*~
+\#*
+*.db
+*.db-journal
+*.doc
+*.docx
+.DS_Store
+/.emacs*
+__MACOSX
+*.out
+*.pdf
+*.pyc
+*.sw*
```

Comment:
I've been writing a temporary toc.md file, and I was considering using a tool such as [cog](https://nedbatchelder.com/code/cog/) to write the table of contents automatically in place. I also generated .docx and .pdf versions for review. Any time I add a .gitignore, I include the standard intermediate files.

Would you prefer to omit the .gitignore file?

Reply:
When I read a .gitignore, for the most part, my expectation is that either:

* the project _does_ generate those files as part of its construction or build pipeline, and they should be ignored; or
* the project _doesn't_ generate those files, but the files are common across many kinds of projects, but are also inconveniently generated by operating systems (e.g. `.DS_Store` from MacOS) or pervasive tooling (e.g. `.tmp` files from editors)

Neither seems to be true in the case of `.docx`, `.pdf`, and most of the others in the list. I don't feel strongly about not changing it, though.

Reply:
Good points. I'll remove the .gitignore since there's no automation in place currently. I added at a time when I was generating several of these files with tools outside the project.